website/docs/r/logz_sub_account.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
+subcategory: "Logz"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_logz_sub_account"
+description: |-
+  Manages a logz Sub Account.
+---
+
+# azurerm_logz_sub_account
+
+Manages a logz Sub Account.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-logz"
+  location = "West Europe"
+}
+
+resource "azurerm_logz_monitor" "example" {
+  name                = "example-monitor"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  plan {
+    billing_cycle  = "MONTHLY"
+    effective_date = "2022-06-06T00:00:00Z"
+    plan_id        = "100gb14days"
+    usage_type     = "COMMITTED"
+  }
+
+  user {
+    email        = "user@example.com"
+    first_name   = "Example"
+    last_name    = "User"
+    phone_number = "+12313803556"
+  }
+}
+
+resource "azurerm_logz_sub_account" "example" {
+  name            = "example-subaccount"
+  logz_monitor_id = azurerm_logz_monitor.example.id
+
+  user {
+    email        = azurerm_logz_monitor.example.user[0].email
+    first_name   = azurerm_logz_monitor.example.user[0].first_name
+    last_name    = azurerm_logz_monitor.example.user[0].last_name
+    phone_number = azurerm_logz_monitor.example.user[0].phone_number
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this logz Sub Account. Changing this forces a new logz Sub Account to be created.
+
+* `logz_monitor_id` - (Required) The ID of the Logz Monitor. Changing this forces a new logz Sub Account to be created.
+
+* `user` - (Required) A `user` block as defined below.
+
+---
+
+* `enabled` - (Optional) Whether the resource monitoring is enabled?
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the logz Sub Account.
+
+---
+
+An `user` block exports the following:
+
+* `email` - (Required) Email of the user used by Logz for contacting them if needed. Changing this forces a new logz Sub Account to be created.
+
+~> **NOTE** If you use the Azure CLI to authenticate to Azure, the Email of your Azure account needs to be granted the admin permission in your Logz.io account. Otherwise, you may not be able to delete this resource. There is no such limitation for the Service Principal authentication.
+
+* `first_name` - (Required) First Name of the user. Changing this forces a new logz Sub Account to be created.
+
+* `last_name` - (Required) Last Name of the user. Changing this forces a new logz Sub Account to be created.
```

Comment:
Can you add a valid value/expected format description here, much like suggested for `name` here?